webservice/model/course.go

```diff
@@ -23,6 +23,38 @@ type Course struct {
 	Assignments []Assignment `json:"assignments"`
 }
 
+// GetCourseCodeAndName gets the course code and name to the course
+func GetCourseCodeAndName(courseID int) (Course, error) {
+	//Create an empty courses array
+	var course Course
+
+	rows, err := db.GetDB().Query("SELECT course.coursecode, course.coursename FROM course WHERE course.id = ?", courseID)
+	if err != nil {
+		log.Println(err.Error())
+
+		// returns empty course array if it fails
+		return course, err
+	}
+
+	for rows.Next() {
+		var courseCode string
```

Comment:
These two strings could be removed with a few changes:
```go
//Create an empty courses array
var course = Course{}

// ...

err := rows.Scan(&course.Code, &course.Name)
```

Reply:
I have to change almost all my db functions, so I'm choosing to to this later since it has no relevance to the Alpha